core/src/test/java/org/elasticsearch/http/netty/NettyHttpChannelTests.java

```diff
@@ -169,16 +175,54 @@ public void testHeadersSet() {
         assertThat(response.headers().get(HttpHeaders.Names.CONTENT_TYPE), equalTo(resp.contentType()));
     }
 
+    public void testLimiterAbortsTooLargeRequests() {
```

Comment:
You're right. After yesterday's changes this test does not make too much sense anymore. I'll remove it.